guide/fr/appendices/file-formats.md

```diff
@@ -93,4 +93,4 @@ For data that changes frequently, and where each pull is limited in size, it is
 
 Like web services, databases provide direct access to data dynamically. Base de donnéess have the advantage that they can allow users to put together just the extraction that they are interested in.
 
-There are some security concerns about allowing remote database extraction and database access is only useful if the structure of the database and the importance of individual tables and fields are well documented. Often, it is relatively simple and inexpensive to create web services that expose data from a database, which can be an easy way to address safety concerns.
\ No newline at end of file
+There are some security concerns about allowing remote database extraction and database access is only useful if the structure of the database and the importance of individual tables and fields are well documented. Often, it is relatively simple and inexpensive to create web services that expose data from a database, which can be an easy way to address safety concerns.
```

Comment:
Hi, i began to translate those pages in French, but small each day... I
will translate next week more and more, as I'm a newbie with Github.
Fortunately , guys in my team can help.
I want to help open data...
;-)

2015-06-04 15:36 GMT+02:00 morchickit <notifications@github.com>:

> In guide/fr/appendices/file-formats.md
> <https://github.com/okfn/opendatahandbook/pull/142#discussion_r31719943>:
>
> > \ No newline at end of file
> > +There are some security concerns about allowing remote database extraction and database access is only useful if the structure of the database and the importance of individual tables and fields are well documented. Often, it is relatively simple and inexpensive to create web services that expose data from a database, which can be an easy way to address safety concerns.
>
> Do you want to keep this in English?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/okfn/opendatahandbook/pull/142/files#r31719943>.
>



-- 

Cordialement,
Patrice Blain-Camara
0611911503